lib/mdb/mdb.js

```diff
@@ -24,7 +24,7 @@ exports.v8ModuleLoaded = function v8ModuleLoaded(output) {
   assert(typeof output === 'string', 'output needs to be a string');
 
   var lines = output.split('\n');
-  if (lines && lines[0].trim().match(/V8 version: \d+\.\d+\.\d+/)) {
+  if (lines && lines[0].trim().match(/(V8|mdb_v8) version: \d+\.\d+\.\d+/)) {
```

Comment:
The output depends on which mdb_v8 version is loaded. With versions < 1.3, I believe mdb_v8 would not output its own version number. 

Moreover, while mdb_v8 might be loaded, it is possible that it fails to get the V8 version information from the core dump being loaded, which would mean that the mdb_v8 version used is incompatible with the version of Node.js being used. So I'm not sure if _just_ checking for the mdb_v8 version is sufficient, and I think that checking for the V8 version is a better indicator of the mdb_v8 module being successfully loaded.

Should we instead be reading the whole set of lines and look for the same V8 version string?

Reply:
Updated PR.  Note that I left the expected version string format the same even though it has 4 levels of versioning and the regex only verifies that there are at least 3.  I figured this would be a more backwards compatible change.